src/lib/common/errorMessages.h

```diff
@@ -76,6 +76,8 @@
 #define ERROR_DESC_BAD_REQUEST_ENTID_IN_PAYLOAD               "entity id specified in payload"
 #define ERROR_DESC_BAD_REQUEST_ENTTYPE_IN_PAYLOAD             "entity type specified in payload"
 #define ERROR_DESC_BAD_REQUEST_NOT_A_JSON_OBJECT              "not a JSON object"
+#define ERROR_DESC_BAD_REQUEST_SUBSCRIPTIONID_NOT_STRING      "/subscriptionId/ must be a JSON String"
```

Comment:
Better to use

...JSON array
...JSON string

to use the sme style in L78 (...JSON object)

Reply:
Fixed in e56e601